src/css/_styles.scss

```diff
@@ -81,30 +72,43 @@ aside {
   width: 200px;
 }
 
-
 /* PANTRY LIST */
 .menu-pantry {
-  align-items: center;
-  background-color: #DAE4EE;
   display: none;
+  background-color: $gray;
   max-height: 60vh;
   overflow: scroll;
-  padding: 12px;
+  padding: 3%;
```

Comment:
I see both hard px and percentages being used for padding. Maybe consider using rem? A percentage might be the solution here, but it's always a good idea to cross browser check since different browsers seem to differ w/ how they calculate %.

https://css-tricks.com/theres-more-to-the-css-rem-unit-than-font-sizing/